lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -471,7 +471,7 @@ def create_build_phase_to_move_static_framework_archive
             build_phase = native_target.new_shell_script_build_phase('Setup Static Framework')
             build_phase.shell_script = <<-eos.strip_heredoc
           mkdir -p "${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.framework/Modules"
-          rsync -t "${BUILT_PRODUCTS_DIR}/lib${PRODUCT_NAME}.a" "${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.framework/${PRODUCT_NAME}"
+          rsync -tL "${BUILT_PRODUCTS_DIR}/lib${PRODUCT_NAME}.a" "${BUILT_PRODUCTS_DIR}/${PRODUCT_NAME}.framework/${PRODUCT_NAME}"
```

Comment:
for posterity can you explain the flag? :)

Reply:
Do any other the `rsync` calls need it?

Reply:
I verified that everything else gets copied. I believe it's related to the construction of the fat library archive that Xcode is using a file link for it.

Reply:
great.